src/scribe_data/utils.py

```diff
@@ -420,3 +420,21 @@ def check_and_return_command_line_args(
         python {all_args[0]} '["comma_separated_sets_in_quotes"]'
         """
     )
+
+def map_genders(wikidata_gender):
+    """
+    Maps those genders from Wikidata to succinct versions.
+
+    Parameters
+    ----------
+        wikidata_gender : str
+            The gender of the noun that was queried from WikiData
+    """
+    if wikidata_gender in ["masculine", "Q499327"]:
+        return "M"
+    elif wikidata_gender in ["feminine", "Q1775415"]:
+        return "F"
+    elif wikidata_gender in ["neuter", "Q1775461"]:
+        return "N"
+    else:
+        return ""  # nouns could have a gender that is not valid as an attribute
```

Comment:
nit:
Add a newline at the end of file

---

There are some uncommon issues that can occasionally occur with the absence of an ending newline 🙃

Reply:
I will do this, thanks.